Source/WebKit/Shared/CoordinatedGraphics/CoordinatedGraphicsScene.h

```diff
@@ -92,6 +99,9 @@ class CoordinatedGraphicsScene : public ThreadSafeRefCounted<CoordinatedGraphics
     // Below two members are accessed by only the main thread. The painting thread must lock the main thread to access both members.
     CoordinatedGraphicsSceneClient* m_client;
     bool m_isActive { false };
+    Vector<WebCore::IntRect> m_lastDamagedRects { };
+    const Vector<WebCore::IntRect> m_noDamagedRects { };
```

Comment:
I think it shouldn't be needed to keep a constant empty copy of a `Vector` here. I would remove this.

Reply:
done